src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -416,6 +416,53 @@ private Locomotive performSwipe(SwipeElementDirection direction, WebElement elem
         return this;
     }
 
+
+    public WebElement swipeTo(SwipeElementDirection s, By by, int attempts) {
+
```

Comment:
Too much vertical spacing, make it consistent with the rest of the code and we are good :)